### PR TITLE
update selection of start and end date

### PR DESCRIPTION
**Pull request by birdage**
update to support annotation generation using user input

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher @Bobfrat review and merge pls

File: ooiui/templates/science/plotting.html [hunk 1]
```diff
@@ -530,16 +537,15 @@
...
+      var min = moment.utc(self.views.svgPlotControlView.model.get('start')).format("DD/MM/YYYY HH:mm");
+      var max = moment.utc(self.views.svgPlotControlView.model.get('end')).format("DD/MM/YYYY HH:mm");
 
       var model = new AnnotationModel({"referenceDesignator":self.models.downloadModel.model.get('reference_designator'),
-                                       "beginDT":min,
-                                       "endDT":max,
+                                       "beginDTSafe":min,
+                                       "endDTSafe":max,
+                                       "beginDT":null,
+                                       "endDT":null,
```
Comment (DanielJMaher) [inline ooiui/templates/science/plotting.html]: What are the points of these "Safe" variables? @birdage

File: ooiui/templates/science/plotting.html [hunk 2]
```diff
@@ -530,16 +537,15 @@
...
     });
 
-    this.listenTo(this, 'AnnotationTableView:onAddClick', function() {
-
-      var bounds = self.views.timeseriesView.views.highchartsView.chart.xAxis[0].getExtremes()
-
-      var min = moment.utc(bounds.userMin).format("YYYY-MM-DDTHH:mm:ss.000")+"Z";
-      var max = moment.utc(bounds.userMax).format("YYYY-MM-DDTHH:mm:ss.000")+"Z";
+    this.listenTo(this, 'AnnotationTableView:onAddClick', function() {      
+      var min = moment.utc(self.views.svgPlotControlView.model.get('start')).format("DD/MM/YYYY HH:mm");
```
Comment (DanielJMaher) [inline ooiui/templates/science/plotting.html]: @birdage Why are we changing the time format from what uFrame accepts?  Does it matter?

**Comment by birdage** [comment]
@DanielJMaher safe does the mapping to and from the uframe/ui versions of datetime

**Comment by DanielJMaher** [comment]
@birdage Makes Sense.

**Pull request merged**